index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Chef Curry</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<nav>
+    <div class="nav-bar">
+        <h1>Chef Curry's Kitchen</h1>
+        <ul class="nav-elements">
+            <a href="index.html">
+            <li>Home</li>
+            </a>
+            <li>About</li>
+        </ul>
+    </div> 
+</nav>
+
+<body>
+    <div class="header-home">
+        <h2>Curry Man's Delights</h2>
+    </div>
+    <div class="recipe-cards">
+        <a href="pani-puri.html">
+        <div class="singular-card">
+            <h3 class="food-label">Pani-Puri</h3>
+            <img src="images/Pani-Puri.jpeg" alt="Pani-Puri">
+            <p>A hollow, crispy-fried puffed ball that is filled with potato, chickpeas, onions, spices, and <br> flavoured water, usually tamarind mint, and popped into one's mouth whole.</p>
+        </div>
+        </a>
+        <a href="dosa.html">
+        <div class="singular-card">
+            <h3 class="food-label">Dosa</h3>
+            <img src="images/dosa.webp" alt="Pani-Puri">
```

Comment:
Try to make the alt text unique to each image (seems like you may have copied it over and forgot to change it)